src/main/java/nl/pim16aap2/armoredElytra/util/ConfigLoader.java

```diff
@@ -86,6 +83,17 @@ private void makeConfig()
                 "If you install additional enchantment plugins, you can add their enchantments as well.",
                 "Just add their 'NamespacedKey'. Ask the enchantment plugin dev for more info if you need it."
             };
+        String[] mutuallyExclusiveEnchantmentsComment =
+            {
+                "The lists of enchantments that are mutually exclusive.",
+                "Each group [] on this list is treated as mutually exclusive, so only one of them can be on an ArmoredElytra.",
+                "The default follows modern vanilla rules by making the different types of protection mutually exclusive.",
+                "If you do not want any enchantments to be mutually exclusive, replace all the entries in this list with \"[]\"",
+                "You can find supported enchantments here:",
+                "https://github.com/PimvanderLoos/ArmoredElytra/blob/master/vanillaEnchantments",
```

Comment:
I'm not really maintaining that list, as it's a bit of a pain with changes/additions between versions. Instead, it might be more convenient to refer users to the console-only  `armoredelytra listAvailableEnchantments` command, which lists all enchantments installed on the server (both vanilla and from other plugins) in the namespace format.

Reply:
Yeah, if you could update that one as well, that'd be great!

Reply:
I've made this change in 6435c825478ed4946fbd73768bcfab1d5a32db5c